storage_service/locations/models/package.py

```diff
@@ -437,6 +438,9 @@ def store_aip(self, origin_location, origin_path):
         # Update package status once transferred to SS
         if dest_space.access_protocol not in (Space.LOM, Space.ARKIVUM):
             self.status = Package.UPLOADED
+        if related_package_uuid is not None:
+            related_package = Package.objects.get(uuid=related_package_uuid)
+            self.related_packages.add(related_package)
```

Comment:
Archivematica does allow you to upload a DIP before storing the AIP, but that is poor practice. We've considered changing this in a future release, by enforcing successful AIP storage before a DIP is allowed to be uploaded.